### ISSUE TITLE
Actually update tests to version 3

### ISSUE THREAD

**Issue by jiegillet** [exercism/elm-test-runner#35]
Currently we mark the `results.json` with a `"version": 3`, but it's not actually earned.
[According to the docs](https://github.com/exercism/docs/blob/main/building/tooling/test-runners/interface.md), version 3 means that the test runner can link individual tests to a task, which we don't do. 

According [to this discussion](https://github.com/exercism/elm-test-runner/issues/30#issuecomment-1056889002), it involves updating the version of `elm-test-rs`. 
It will also involve a large PR in `exercism/elm` to tag all the concept exercise tests to their task.

**Comment by jiegillet** [comment]
I forgot to mention that the UI for showing the task ID is live now.

<img width="482" alt="Screen Shot 2022-03-26 at 21 09 33" src="https://user-images.githubusercontent.com/16929078/160238747-4829ae61-f2cd-45a2-a4e9-319075c16ccb.png">

**Comment by mpizenberg** [comment]
> I forgot to mention that the UI for showing the task ID is live now

:tada:

**Comment by mpizenberg** [comment]
Really nice to have the UI for tasks :)
My bad, must be my mistake if I updated the version to 3 while this was meant for tests categorized by tasks. Or the version 3 spec changed at some point to include those.
The good thing about still having a relatively small learning track is that it won't be that big to update a few tests files. Usually, tests are already close together for the same task. So it will only be a matter of transforming tests descriptions like this:
```elm
-- before
describe "concept"
  [ test "test 1a" <| ...
  , test "test 1b" <| ...
  , test "test 2a" <| ...
  ]

-- after
describe "concept"
  [ describe "1" -- the task id
    [ test "test 1a" <| ...
    , test "test 1b" <| ...
    ]
  , describe "2"
    [ test "test 2a" <| ...
    ]
  ]
```

**Comment by jiegillet** [comment]
Closed via #36.
